demisto_sdk/commands/validate/validators/GR_validators/GR101_is_using_invalid_from_version_all_files.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import annotations
+
+from typing import Iterable, List, Union
+
+from demisto_sdk.commands.common.constants import ExecutionMode
+from demisto_sdk.commands.content_graph.objects import (
+    CaseField,
+    CaseLayout,
+    CaseLayoutRule,
+    Classifier,
+    CorrelationRule,
+    Dashboard,
+    GenericDefinition,
+    GenericField,
+    GenericModule,
+    GenericType,
+    IncidentType,
+    IndicatorField,
+    IndicatorType,
+    Integration,
+    Job,
+    Layout,
+    LayoutRule,
+    Mapper,
+    ModelingRule,
+    Pack,
+    ParsingRule,
+    Playbook,
+    Report,
+    Script,
+    TestPlaybook,
+    Trigger,
+    Widget,
+    Wizard,
+    XSIAMDashboard,
+    XSIAMReport,
+)
+from demisto_sdk.commands.content_graph.objects.incident_field import IncidentField
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+from demisto_sdk.commands.validate.validators.GR_validators.GR101_is_using_invalid_from_version import (
+    IsUsingInvalidFromVersionValidator,
+)
+
+ContentTypes = Union[
```

Comment:
I'm surprised that you need it even though it's already in the 'father' file. Can you try to understand why?```suggestion

```